index.html

```diff
@@ -18,13 +18,13 @@ <h1>Launch Checklist Form</h1>
                     <label>Pilot Name <input type="text" name="pilotName" id="pilotName"/></label>
                 </div>
                 <div class="formField">
-                    <label>Co-pilot Name <input type="text" name="copilotName"/></label>
+                    <label>Co-pilot Name <input type="text" name="copilotName" id="copilotName"/></label>
```

Comment:
```suggestion
                    <label>Co-pilot Name <input type="text" name="copilotName"/></label>
```
Good job using the `id` attribute to get the elements! For this assignment, you can also get the elements with the attribute selector with `querySelector` as well:

document.querySelector(input[name='copilotName']);